typed_python/types_test.py

```diff
@@ -171,7 +171,7 @@ def pickRandomly(self):
 
 class NativeTypesTests(unittest.TestCase):
 
-    def check_expected_performance(self, elapsed, expected=1.0):
+    def check_expected_performance(self, elapsed, expected=1.5):
```

Comment:
rather than widening this out, can you try setting the TRAVIS_CI environment variable and running tests that way? We have widened out all test run requirements for travis already so that should work for you everywhere..